include/mbedtls/block_cipher.h

```diff
@@ -0,0 +1,58 @@
+/**
+ * \file block_cipher.h
+ *
+ * \brief Internal abstraction layer.
```

Comment:
Internal? Being in `include/mbedtls.h` does not make it public?
What about using the same description of `block_cipher_internal.h`?

Reply:
Only the types are public, because we'll need to use them in `gcm.h` and `ccm.h` which are public. You can't do anything useful with just the public header, so I'm saying "internal" and not giving any detail as a way of saying there's nothing to see here for normal users - this is only "public" for the compiler.

Do you have suggestions for a better way to reflect this?

Reply:
Oh, OK, thanks for the explanation. That's OK for me then